packages/Button/src/Button.spec.tsx

```diff
@@ -16,8 +16,20 @@ describe('packages/Button', () => {
     </Button>,
   );
 
+  function isElement(el: Node): el is HTMLElement {
+    return el.nodeType === Node.ELEMENT_NODE;
+  }
+
+  function isButton(el: Node): el is HTMLButtonElement {
+    return isElement(el) && el.tagName.toLowerCase() === 'button';
+  }
+
   const renderedButton = renderedComponent.container.firstChild;
 
+  if (renderedButton == null || !isElement(renderedButton)) {
```

Comment:
There a reason why we aren't simply checking this?

```js
if (!isElement(renderedButton)) {
```

`isElement` should return false here if `renderedButton` is nullish. Does TS complain that `renderedButton` could be nullish and doesn't match the `Node` type?

Reply:
If I had defined `isElement` as taking `Node | null` then that'd be fine. I guess that's probably a more useful typeguard defined that way, I'll change it.